resources/scheduled_task.rb

```diff
@@ -19,12 +19,12 @@
 
 resource_name :chef_client_scheduled_task
 
-property :user, String, default: 'System'
+property :user, String, default: node['chef_client']['task']['user']
```

Comment:
These node attributes won't actually work if someone overrides them in a role/env/cookbook.